webapp/config/pagination.py

```diff
@@ -0,0 +1,15 @@
+from collections import OrderedDict
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+
+class PageNumberPagination(PageNumberPagination):
+    page_size = 3
+    # 임시 설정
```

Comment:
```suggestion
    page_size = 10
```
p2: 10개정도가 무난할 것 같슴다